pyccl/__init__.py

```diff
@@ -23,6 +23,9 @@
 # SWIG-generated
 from . import ccllib as lib
 
+# Name of the default power spectrum.
+DEFAULT_POWER_SPECTRUM = "delta_matter:delta_matter"
```

Comment:
I think I'd rather have "delta_matter:delta_matter" everywhere, rather than this. It's easier for users to understand from the documentation and function signature.